domains/certificates/chart/templates/postgres-statefulset.yaml

```diff
@@ -15,29 +19,49 @@ spec:
       labels:
         app: {{ .Release.Name }}-postgres
     spec:
-      volumes:
-        - name: postgres-vm
-          configMap:
-            name: postgres-init-cm
       containers:
         - name: {{ .Release.Name }}-postgres
           image: '{{ .Values.postgres.image.name }}:{{ .Values.postgres.image.tag}}'
-          volumeMounts:
-            - name: postgres-vm
-              mountPath: /docker-entrypoint-initdb.d/
-              readOnly: true
+          ports:
+            - containerPort: 5432
+              hostPort: 5432
+              name: pgsql
+              protocol: TCP
           readinessProbe:
             tcpSocket:
               port: 5432
             initialDelaySeconds: 2
             periodSeconds: 2
           livenessProbe:
-            tcpSocket:
-              port: 5432
-            initialDelaySeconds: 2
-            periodSeconds: 2
-          ports:
-            - containerPort: 5432
-              hostPort: 5432
-              name: pgsql
-              protocol: TCP
+            failureThreshold: 6
+            initialDelaySeconds: 30
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 5
+            exec:
+              command:
+                - /bin/sh
+                - -c
+                - exec pg_isready -U "postgres" -h 127.0.0.1 -p 5432
+          volumeMounts:
+            - name: dshm
+              mountPath: /dev/shm
+            - name: postgres-vm
+              mountPath: /docker-entrypoint-initdb.d/
+              readOnly: true
+      volumes:
+        - name: dshm
+          emptyDir:
+            medium: Memory
+        - name: postgres-vm
+          configMap:
+            name: postgres-init-cm
+  volumeClaimTemplates:
+    - metadata:
+        name: data
```

Comment:
@CodeReaper This is not referenced anywhere. Is it used? And if yes, where is it used?